app/models/manageiq/providers/kubernetes/container_manager.rb

```diff
@@ -544,6 +534,20 @@ def self.params_for_create
     }
   end
 
+  def virtualization_options
```

Comment:
`.params_for_create` is a class method so this has to be also 

```suggestion
  def self.virtualization_options
```